entrypoint.sh

```diff
@@ -14,8 +14,9 @@ replace_in_files() {
         VALUE=`printenv "$VAR_NAME"`
 
         # Replace the variable only if it starts with the name of the directive and remove optional ';'
+        # Some directives will not have spaces after the directive name. i.e opcache
         find $PHP_CONFIG_FILES -type f -exec \
-            sed -i "s/^\(;\)\{0,1\}$DIRECTIVE = [^\n]\+/$DIRECTIVE = $VALUE/g" {} \;
+            sed -i "s/^\(;\)\{0,1\}$DIRECTIVE\( \)\{0,1\}=.*\+/$DIRECTIVE=$VALUE/g" {} \;
```

Comment:
Why are we matching newlines now? That will potentially cause this to replace more than just the directive we are targeting.

Reply:
Just marking the spaces in the existing regex optional should address the problem:

`sed -i "s/^\(;\)\{0,1\}$DIRECTIVE ?= ?[^\n]\+/$DIRECTIVE = $VALUE/g" {} \;`

Reply:
I was able to use `?` I needed to scape it. and worked as expected. 

```
$ docker build -f php.docker .   
Successfully built 277cfde754b0
$ docker run -e PHP_ALL_OPCACHE__FILE_CACHE="\/var\/cache\/php\/opcache" 277cfde754b0 cat /etc/php/7.3/fpm/php.ini | grep "opcache\."
opcache.file_cache=/var/cache/php/opcache
```